packages/base/catalog-entry.gts

```diff
@@ -3,23 +3,26 @@ import {
   field,
   Component,
   CardDef,
-  FieldDef,
   relativeTo,
   realmInfo,
+  linksToMany,
 } from './card-api';
 import StringField from './string';
 import BooleanField from './boolean';
 import CodeRef from './code-ref';
+import MarkdownField from './markdown';
 
-import { FieldContainer } from '@cardstack/boxel-ui/components';
 import GlimmerComponent from '@glimmer/component';
 import BoxModel from '@cardstack/boxel-icons/box-model';
+import BookOpenText from '@cardstack/boxel-icons/book-open-text';
+import LayersSubtract from '@cardstack/boxel-icons/layers-subtract';
+import GitBranch from '@cardstack/boxel-icons/git-branch';
 
 export class CatalogEntry extends CardDef {
   static displayName = 'Catalog Entry';
   static icon = BoxModel;
-  @field title = contains(StringField);
-  @field description = contains(StringField);
+  @field readMe = contains(MarkdownField);
+
   @field ref = contains(CodeRef);
```

Comment:
the embedded view for this field type is super bare bones. could use a little CSS love.